### DISCUSSION TITLE
Kommastelle  Einheit für Gesamtertrag

### DISCUSSION THREAD

**Discussion by daytona80** [tbnobody/OpenDTU#2087] [General]
Würde gern von kWh auf MWh für den Gesamtertrag umstellen, da im 1Zoll Display zu wenig Platz für die Anzeige ist. Ja, hab schon über 10MWh Ertrag ;)

**Comment by Solarteur** [comment]
Ich habe zwar kein Platzproblem auf dem Display, würde aber vorschlagen, für "Inverter Gesamtertrag Insgesamt" die Nachkommastellen nicht anzuzeigen und bei kWh zu bleiben. Die Nachkommastellen gaukeln eine Genauigkeit vor, die in der Praxis nicht gegeben ist.

**Comment by daytona80** [comment]
Das mit den Nachkommastellen wäre auch hilfreich und 3 Stellen Platz schaffen. Da ich nach 1 Jahr schon mehr als 10MWh habe wird es perspektivisch in 10J dann "nur noch" 1 Stelle nehr als jetzt...
